### PR TITLE
Fixes #19497 - More passenger info in foreman-debug

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

**Comment by sean797** [comment]
test failure is unrelated.

File: script/foreman-debug [hunk 1]
```diff
@@ -343,6 +343,8 @@ add_files /etc/{sysconfig,default}/libvirt*
 add_files /etc/sysconfig/pgsql
 add_files /var/lib/pgsql/data/pg_log/*
 add_cmd "foreman-rake plugin:list" "plugin_list"
+add_cmd "passenger-status -v"
+add_cmd "passenger-memory-stats"
```
Comment (xprazak2) [inline script/foreman-debug]: I think you need to specify output file:

```bash
add_cmd "passenger-status -v" "passenger_status"
add_cmd "passenger-memory-stats" "passenger_memory"
```

**Comment by sean797** [comment]
Thanks for looking @xprazak2. sorry - yes you do. Updated.

**Comment by ares** [comment]
Ready after Mr. Jenkins confirms.

**Comment by lzap** [comment]
Wait, what's "-v" for? I can only see difference in extra line containing "password".

**Comment by lzap** [comment]
Yeah please remove "-v" from there, it exposes passenger API alonside with password, we do not want that.

Consider adding these outputs too: `--show=pool|requests|backtraces`

**Comment by sean797** [comment]
Updated. I thought foreman-debug filtered out strings like "password" but reading it, it appears to only do so for `add_file` not `add_cmd`.

Thanks all!

**Comment by lzap** [comment]
There is no point in waiting for jenkins, no tests are present for debug. Thanks @sean797 !

**Pull request merged**